cloudflared/umbrel-app.yml

```diff
@@ -0,0 +1,35 @@
+manifestVersion: 1
+id: cloudflared
+name: Cloudflare Tunnel client
+tagline: Access your Umbrel apps from the Internet using Cloudflare network
+icon: https://raw.githubusercontent.com/Radiokot/umbrel-cloudflared/main/static/logo.svg
+category: Networking
+version: "2023.10.0 (web 1.0.1)"
+port: 4499
+description: >-
+  Start a secure tunnel to access your Umbrel apps from the Internet using Cloudflare network. 
+  With Tunnel, you do not send traffic to an external IP — instead, 
+  this app containing a lightweight tunneluing daemon (cloudflared) creates outbound-only 
+  connections to Cloudflare's global network.
+
+
+  To use this app, you must have a set up Cloudflare account with added domains. 
+  Once you've done setting it up, check out the guide with examples to configure your own tunnel:
+  https://github.com/Radiokot/umbrel-cloudflared/wiki/How-to-set-up-Cloudflare-Tunnel-on-your-Umbrel
+
+developer: Cloudflare
```

Comment:
You created an amazing frontend for this app, so I'm thinking this should be you:
```yaml
developer: Radiokot
```

What are your thoughts on placing you as the app developer, but then at the end of the app description saying something like "Powered by Cloudflare" like we do for apps like Lightning Node:

https://github.com/getumbrel/umbrel-apps/blob/f6eae6fdc53a1bd71528fed49118e299da5d1aea/lightning/umbrel-app.yml#L7-L20